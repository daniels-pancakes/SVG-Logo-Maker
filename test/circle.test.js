const inquirer = require('inquirer');
const shape = require('../lib/shapes.js');

describe('Shape', () => {
    describe('Circle', () => {
        describe('Render', () => {
        it('should output code to build an SVG xml file.', () => {
            const circle = new shape.Circle();
            const svg = circle.render('text', 'textcolor', 'shapecolor');
            expect(svg).toMatchSnapshot();
        });
    })
    })
});