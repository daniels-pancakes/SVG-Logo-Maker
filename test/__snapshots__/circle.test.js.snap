// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shape Circle Render should output code to build an SVG xml file. 1`] = `
"<svg viewBox="0 0 50 50" xmlns="http://www.w3.org/2000/svg">
        <defs>
            <style type="text/css">
            </style>
        </defs>
            <circle cx="25" cy="25" r="20" 
            fill="shapecolor"
            />
            <text fill="textcolor" text-anchor="middle" dominant-baseline="middle" x="25" y="26.5" class="text">text</text>
            <text style="font-size:2; letter-spacing:1.5;" fill="textcolor" text-anchor="middle" dominant-baseline="middle" x="26" y="35.5" class="text">EST. 2024</text>
        </svg>"
`;
